multiples.js

```diff
@@ -7,6 +7,20 @@ exports.sumOfAMultiple = function( n ) {
   let sum = 0;
 
   // do your work here
-  
+
+  let newArr = [];
+
+  for(i = 0; i < n; i++) {
+    if(i % 3 === 0) {
+      newArr.push(i)
+    } else if(i % 5 === 0) {
+      newArr.push(i);
+    }
+  }
+
+  for(i = 0; i < newArr.length; i++) {
```

Comment:
Instead of putting all the values into an array, you could do the summation on the fly in the for loop.